config/settings.yaml.example

```diff
@@ -23,5 +23,14 @@
 :domain: 'localdomain.net'
 :fqdn: 'localhost.localdomain.net'
 
-# Should ANSI color codes be used in logs
-:colorize_logging: false
+# Loggers can be configured individually, they can be turned off completely by using key
+# 'enabled' or you can adjust its logging 'level'. If you specify a level higher than root
+# logger has, messages below this level are ignored. If you specify lower level than root
+# they are still ignored but you can add appender in Foreman::Logging just for this logger
+# and see these logs outside of main log.
+:loggers:
+  :app:
+    :enabled: true
+  :sql:
+    :enabled: true
+    :level: fatal
```

Comment:
any need for this?  I'm trying to think if there's any AR logging we're trying to hide between info and fatal level, and can't think of anything - it's mostly all debug.